### PR TITLE
feat: dark mode

### LINKED ISSUE #71: Darkmode
### textarea

I have a darkmode chrome extension and the site looks really nice 

Screenshot 2024-04-30 at 12 09 57 AM

### PR DESCRIPTION
**Pull request by lospoy**
Adds:
- dark mode (using the dark background for all pages)
- `<Card>` component

Modifies:
- /links page now uses Tailwind and `<Card>` 
- `useWavyBackground` is now darker
- `FeedItem` component now uses `<Card>`
- `MainLayout` style update for small vs large screens

Staging:
https://lospoy.github.io/meetup/

Closes #71

### CODE REVIEW

**Comment by lospoy** [comment]
@reggi sorry about the commit history, don't understand why the branch has 2 old commits :person_facepalming:

**Comment by reggi** [comment]
* What do you think about having consistency between home and links? 
* From a design standpoint why would we have two backgrounds? (I'd like to just use the dark one.)
* For the `FeedItem` I like the background and border, I think that should be on links page too. We'd need a shared component for `<Card>`.
* You're implementing darkmode and removing light mode. and not using tailwinds builtins for [dark](https://tailwindcss.com/docs/dark-mode), is there a reason? [See my personal site for example with button to switch between to two](https://reggi.com/).

File: src/layouts/Card.astro [hunk 1]
```diff
@@ -0,0 +1,7 @@
+---
+
+---
+
+<div class="mb-5 dark:bg-[#2d272e] px-5 py-7 dark:text-[#f8fafc] dark:border-[#1c1917] border-8 rounded-2xl">
+  <slot />
+</div>
```
Comment (reggi) [inline src/layouts/Card.astro:7]: 👋 this is a component not a layout, layouts have like `<html/>` and typically wrap a page.

**Review by reggi — CHANGES_REQUESTED** [review]
looks good just need to move card to components

**Review by reggi — APPROVED** [review]

**Comment by reggi** [comment]
Looks great!

**Pull request merged**